products/BellHybrid/services/db/databases/migration/settings_bell/1/up.sql

```diff
@@ -1,10 +1,14 @@
 -- Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 -- For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
+-- Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+-- For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
```

Comment:
please remove the additional copyright info. 
@swichu91 do you have any clue why that happened?

Reply:
It seems like a minor issue in `db_migration.py` script. I will have a look at it.